pgjdbc/src/test/java/software/aws/rds/jdbc/postgresql/auth/AwsIamAuthenticationPluginTest.java

```diff
@@ -0,0 +1,105 @@
+package software.aws.rds.jdbc.postgresql.auth;
+
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.when;
+
+import org.postgresql.PGProperty;
+import org.postgresql.plugin.AuthenticationRequestType;
+import org.postgresql.util.PSQLException;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import software.amazon.awssdk.regions.Region;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+import java.time.Instant;
+import java.util.Properties;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class AwsIamAuthenticationPluginTest {
+
+  @Test
+  public void testGetPasswordWithWrongType() throws PSQLException {
+    Properties properties = new Properties();
```

Comment:
The first 4 lines are repeated in every test.

Can you check on how to use @Before and @BeforeEach and remove those from the test code?